tests/pykafka/test_ssl.py

```diff
@@ -0,0 +1,64 @@
+import unittest
+from uuid import uuid4
+
+import pytest
+
+from pykafka import KafkaClient, SslConfig
+from pykafka.test.utils import get_cluster, stop_cluster
+
+
+class SslIntegrationTests(unittest.TestCase):
+    USE_RDKAFKA = False
+
+    @classmethod
+    def setUpClass(cls):
+        cls.kafka = get_cluster()
+        if cls.kafka.brokers_ssl is None:
+            pytest.skip("Test-cluster doesn't advertise ssl ports.")
+
+    @classmethod
+    def tearDownClass(cls):
+        stop_cluster(cls.kafka)
+
+    def roundtrip_test(self, client):
```

Comment:
Why does this use a naming convention that keeps it from being automatically run in travis?